assets/css/team.css

```
@@ -1,231 +1,414 @@
+    flex-direction: column;
+    align-items: center;
 }
 
-.member-image img.loaded {
-  opacity: 1;
+.s-team__desc {
+    text-align: center;
+    margin-bottom: 1.5rem;
+    width: 100%;
+    max-width: 100%; /* Full width container */
+    margin: 0 auto;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)